docs: add single session per user docs (#19411)
supabase/supabase: apps/docs/pages/guides/auth/sessions.mdx

- The user changes their password or performs a security sensitive action.
- It times out due to inactivity.
- It reaches its maximum lifetime.

> - A user signed in on another device.

Reviewer: ```suggestion
- A user signs in on another device.
```

I'd suggest putting the condition here (`when "single user per session" is configured` or something like that), as it's pretty easy to misread this as the default behavior, which might not be what many people want. (Might apply to the other items as well)

Author: It has a sentence above it that says "depending on configuration"

Be aware that access tokens of revoked sessions remain valid until their expiry time, encoded in the `exp` claim.

> ## Controlling the lifetime and number of sessions a user can have

Reviewer: ```suggestion
## Limiting session lifetime and number of allowed sessions per user
```

To make sure that sessions expire after a period of inactivity, you can set a positive duration for the _Inactivity timeout_ option in the [Auth settings](/dashboard/project/_/settings/auth).

> You can also configure to have only one active session per user per device or browser. When this is enabled, the session from the most recent sign in will remain active, while the rest are terminated. Enable this via the _Single session per user_ option in the [Auth settings](/dashboard/project/_/settings/auth).

Reviewer: ```suggestion
You can also enforce only one active session per user per device or browser. When this is enabled, the session from the most recent sign in will remain active, while the rest are terminated. Enable this via the _Single session per user_ option in the [Auth settings](/dashboard/project/_/settings/auth).
```
